src/docs/1.0/getting-started.html

```diff
@@ -9,7 +9,20 @@
 	<link rel="stylesheet" href="../../css/style.min.css">
 </head>
 <body>
-	
+	<div class="component-container">
+		<h2>Alerts</h2>
+		<!-- primary alert -->
+		<div class="alert alert-primary">Primary Alert</div>
+		<!-- secondary alert -->
+		<div class="alert alert-secondary">Secondary Alert</div>
+		<!-- success alert -->
+		<div class="alert alert-success">Success Alert</div>
+		<!-- danger/error alert -->
+		<div class="alert alert-error">Danger/Error Alert</div>
+		<!-- warning alert -->
+		<div class="alert alert-warning">Warning Alert</div>
+		<pre><code class="language-html">&lt;!-- primary alert --&gt;&#10;&lt;div class=&quot;alert alert-primary&quot;&gt;Primary Alert&lt;/div&gt;&#10;&lt;!-- secondary alert --&gt;&#10;&lt;div class=&quot;alert alert-secondary&quot;&gt;Secondary Alert&lt;/div&gt;&#10;&lt;!-- success alert --&gt;&#10;&lt;div class=&quot;alert alert-success&quot;&gt;Success Alert&lt;/div&gt;&#10;&lt;!-- danger/error alert --&gt;&#10;&lt;div class=&quot;alert alert-error&quot;&gt;Danger/Error Alert&lt;/div&gt;&#10;&lt;!-- warning alert --&gt;&#10;&lt;div class=&quot;alert alert-warning&quot;&gt;Warning Alert&lt;/div&gt;</code></pre>
```

Comment:
Even adding "code" element in new line is giving extra white space around code snippet in the output, because with usage of "pre" element it is rendering as it is in the output